scifight/templatetags/scifight_url.py

```diff
@@ -0,0 +1,53 @@
+from django import template
+from scifight import models
+from django.core.urlresolvers import reverse, NoReverseMatch
+
+
+register = template.Library()
+
+_single_arg_views = {
+    models.Participant: ("scifight:participant",   'id'),
+    models.Leader:      ("scifight:leader",        'id'),
+    models.Jury:        ("scifight:jury",          'id'),
+    models.Room:        ("scifight:room",          'id'),
+    models.Fight:       ("scifight:fight",         'id'),
+    models.Problem:     ("scifight:problem",       'problem_num')
+}
+
+
+@register.filter(is_safe=True)
+def scifight_url(model):
+    """
+    This filter simplifies the use of a template.
+    Input parameter is a model object.
+    {{ object | scifight_url }}
+    The filter will return the URL, from the corresponding
+    item in the urls.py. In case of an error NoReverseMatch
+    returns an empty string
+    """
+    def wrapped_reverse(*args, **kwargs):
+        try:
+            return reverse(*args, **kwargs)
+        except NoReverseMatch:
+            return ''
+
+    if isinstance(model, models.Team):
```

Comment:
In the code below, we use type equality checks instead of `isinstance`. Let's do it here too just for the sake of consistency. So, the code would look like the following:
```python
model_type = type(model)
if model_type == models.Team:
    return ...

if model_type == models.Tournament:
    return ...

...
```